tests/app/celery/test_tasks.py

```diff
@@ -861,26 +872,44 @@ def test_process_rows_sends_save_task(
                 "to": "recip",
                 "row_number": "row_num",
                 "personalisation": {"foo": "bar"},
-                "queue": None,
+                "queue": "send-{}-tasks".format(template_type),
                 "client_reference": reference,
                 "sender_id": str(sender_id) if sender_id else None,
             }
         )
         task_mock.apply_async.assert_called_once()
 
     @pytest.mark.parametrize(
-        "csv_threshold, expected_queue",
+        "csv_normal_threshold, csv_bulk_threshold, template_process_type, expected_queue",
```

Comment:
can 🔥 the `csv_normal_threshold` here and in the test

Reply:
removing...

Reply:
This was removed. 👍